### PR TITLE
Integrate sabre/cs

### PR DESCRIPTION
**Pull request by evert**

### CODE REVIEW

File: tests/HTTP/Auth/DigestTest.php [hunk 1]
```diff
@@ -168,15 +167,15 @@ private function getServerTokens($qop = Digest::QOP_AUTH) {
         $this->auth->requireLogin();
 
         switch($qop) {
-            case Digest::QOP_AUTH    : $qopstr='auth'; break;
-            case Digest::QOP_AUTHINT : $qopstr='auth-int'; break;
-            default                                 : $qopstr='auth,auth-int'; break;
+            case Digest::QOP_AUTH    : $qopstr = 'auth'; break;
+            case Digest::QOP_AUTHINT : $qopstr = 'auth-int'; break;
+            default                                 : $qopstr = 'auth,auth-int'; break;
```
Comment (Hywan) [inline tests/HTTP/Auth/DigestTest.php]: The `:` is faaaaaaaaaaar away :stuck_out_tongue:.

**Comment by Hywan** [comment]
You run `sabre-cs-fixer` on `lib/` or `lib/` + `test/`, because the `test/` directory has been updated too.

File: tests/HTTP/Auth/DigestTest.php (hunk 1 above)
Comment (evert) [inline tests/HTTP/Auth/DigestTest.php]: Hmm yes weird.

Comment (evert) [inline tests/HTTP/Auth/DigestTest.php]: Oh it already was in the past =)

**Comment by evert** [comment]
Yep I also used it for `test/`!

**Pull request merged**